packages/react-components/react-avatar/src/components/Avatar/useAvatarStyles.ts

```diff
@@ -378,24 +378,27 @@ const useColorStyles = makeStyles({
 export const useAvatarStyles_unstable = (state: AvatarState): AvatarState => {
   const { size, shape, active, activeAppearance, color } = state;
 
+  const rootClassName = useRootClassName();
+  const imageClassName = useImageClassName();
+  const iconInitialsClassName = useIconInitialsClassName();
   const styles = useStyles();
   const sizeStyles = useSizeStyles();
   const colorStyles = useColorStyles();
 
-  const rootClasses = [styles.root, sizeStyles[size], colorStyles[color]];
+  const rootClasses = [rootClassName, size !== 32 && sizeStyles[size], colorStyles[color]];
```

Comment:
You could alternatively alter line 202 to be

`32: { /* Default size in useRootClassName */ },`

and leave this line as

```suggestion
  const rootClasses = [rootClassName, sizeStyles[size], colorStyles[color]];
```

Reply:
Those styles are used in other avatar controls so it's simpler to leave it as-is.